titus-server-master/src/main/java/com/netflix/titus/master/config/MasterConfiguration.java

```diff
@@ -92,4 +92,8 @@
     @PropertyName(name = "mesos.slave.attribute.zone.name")
     @DefaultValue("zone")
     String getHostZoneAttributeName();
+
+    @PropertyName(name = "titus.jobcoordinator.pod.containerInfoEnvEnabled")
+    @DefaultValue("true")
+    boolean isContainerInfoEnvEnabled();
```

Comment:
Minor: could/should this be a `FeatureRolloutPlans`?

Reply:
Thanks for the feedback. I am hoping to activate this change before the holidays. Hopefully, we won't need feature plan for this short term rollout! Also, it felt the feature plan evaluations are used more for surfacing violation messages to the user? Lmk if I am missing something.